bip-0077.mediawiki

```diff
@@ -0,0 +1,286 @@
+
+The sender constructs an encrypted and authenticated payload containing a PSBT and optional parameters similar to BIP 78. The resulting ciphertext ensures message secrecy and integrity when passed to the receiver by the subdirectory <code>pj=</code> endpoint.
+
+Messages are secured by symmetric cipher rather than TLS or Onion routing session key. Sender and receiver may experience network interruption and proceed with the protocol since their request and response are buffered at the payjoin directory.
+
+====Sequence Diagram====
+
+<pre>
++----------+                 +-----------+       +--------+  +---------+
+| Receiver |                 | Directory |       | Sender |  | Network |
++----------+                 +-----------+       +--------+  +---------+
+|                                  |                  |                |
+| Request subdirectory             |                  |                |
++--------------------------------->|                  |                |
+|                                  |                  |                |
+|           subdirectory           |                  |                |
+|<---------------------------------|                  |                |
+|                                  |                  |                |
+|      destination (address,       |                  |                |
+|        subdirectory)             |                  |                |
++---------------------------------------------------->|                |
+|                                  |                  |                |
+|      Request Original PSBT       |                  |                |
++- - - - - - - - - - - - - - - - ->|                  |                |
+|                                  |  Original PSBT   |                |
+|                                  |<- - - - - - - - -|                |
+|                                  |                  |                |
+|                                  |                  |                |
+|          Original PSBT           |                  |                |
+|<---------------------------------|                  |                |
+|                                  |                  |                |
+|                                  |                  |                |
+|             Payjoin PSBT         |                  |                |
++--------------------------------->|                  |                |
+|                                  |   Payjoin PSBT   |                |
```

Comment:
It does poll for the Payjoin PSBT by sending a payload containing the Original PSBT

Reply:
Got it. I might consider adding a footnote for that. The way the diagram is, it looks like the directory pushes a Payjoin PSBT to the sender. But I also think you are quite limited by what you can do with this format so instead of complicating the diagram, it may just be something that could be mentioned at the bottom.